src/frontend/src/pages/ChangeRequestDetailPage/ChangeRequestDetailsView.tsx

```diff
@@ -124,70 +153,104 @@ const ChangeRequestDetailsView: React.FC<ChangeRequestDetailsProps> = ({
+      />
+
+      <NERButton
+        sx={{ mt: '20px', float: 'right' }}
+        variant="contained"
+        disabled={!isRequestAllowed}
+        onClick={handleRequestReviewerClick}
+      >
+        Request Review
+      </NERButton>
     </div>
   );
 
-  const implementCrDropdown = (
+  const unreviewedActionsDropdown = (
     <div>
       <NERButton
         endIcon={<ArrowDropDownIcon style={{ fontSize: 28 }} />}
         variant="contained"
-        id="implement-cr-dropdown"
+        id="unreviewed-cr-actions-dropdown"
         onClick={handleClick}
       >
-        Implement Change Request
+        Actions
       </NERButton>
-      <Menu
-        open={dropdownOpen}
-        anchorEl={anchorEl}
-        onClose={handleDropdownClose}
-        anchorOrigin={{
-          vertical: 'bottom',
-          horizontal: 'right'
-        }}
-        transformOrigin={{
-          vertical: 'top',
-          horizontal: 'right'
-        }}
-      >
-        <MenuItem
-          to={`${routes.PROJECTS_NEW}?crId=${changeRequest.crId}&wbs=${projectWbsPipe(changeRequest.wbsNum)}`}
-          component={RouterLink}
-          onClick={handleDropdownClose}
-          disabled={!isUserAllowedToImplement}
-        >
-          <ListItemIcon>
-            <CreateNewFolderIcon fontSize="small" />
-          </ListItemIcon>
-          Create New Project
-        </MenuItem>
-        <MenuItem
-          component={RouterLink}
-          to={`${routes.WORK_PACKAGE_NEW}?crId=${changeRequest.crId}&wbs=${projectWbsPipe(changeRequest.wbsNum)}`}
-          disabled={!isUserAllowedToImplement}
-          onClick={handleDropdownClose}
-        >
-          <ListItemIcon>
-            <PostAddIcon fontSize="small" />
-          </ListItemIcon>
-          Create New Work Package
+      <Menu open={dropdownOpen} anchorEl={anchorEl} onClose={handleDropdownClose}>
+        <MenuItem onClick={handleReviewOpen} disabled={!isUserAllowedToReview}>
+          Review
         </MenuItem>
-        <MenuItem
-          component={RouterLink}
-          to={`${routes.PROJECTS}/${wbsPipe(changeRequest.wbsNum)}?crId=${changeRequest.crId}&edit=${true}`}
-          disabled={!isUserAllowedToImplement}
-          onClick={handleDropdownClose}
-        >
-          <ListItemIcon>
-            <EditIcon fontSize="small" />
-          </ListItemIcon>
-          Edit {changeRequest.wbsNum.workPackageNumber === 0 ? 'Project' : 'Work Package'}
+        <Divider />
+        <MenuItem disabled={!isUserAllowedToDelete} onClick={handleDeleteOpen}>
+          Delete
         </MenuItem>
       </Menu>
     </div>
   );
 
-  const actionDropdown = changeRequest.accepted ? implementCrDropdown : unreviewedActionsDropdown;
+  const renderUnreviewedActionsDropdown =
+    isRequestAllowed && changeRequest.status === ChangeRequestStatus.Open
+      ? openedActionsDropdown
+      : unreviewedActionsDropdown;
+
+  const implementCrDropdown = (
+    <div>
+      <ActionsMenu
+        buttons={[
```

Comment:
How would I add **to** and **component** fields' function for each button?

For ex, 

```
to={`${routes.PROJECTS_NEW}?crId=${changeRequest.crId}&wbs=${projectWbsPipe(changeRequest.wbsNum)}`}
component={RouterLink}
```

for `Create New Project` button.

Reply:
I think from ActionsMenu component definition, we have to add router's `to` feature in the OnClick field? I wonder what we use to navigate our page to certain links.